src/modules/createList.js

```diff
@@ -33,6 +34,26 @@ const populate = async (movies) => {
     `;
     }
   });
+
+  const likebtn = Array.from(document.querySelectorAll('.my-btn1'));
+
+  await likebtn.forEach(async (btn) => {
+    const btnID = btn.id;
+    await movies.forEach(async (elem, index) => {
+      const iD = `like${index}`;
+      if (btnID === iD) {
+        await btn.addEventListener('click', async () => {
+          addLike(elem.imdbid);
+          const likeArr = await displayLikes();
+          likeArr.forEach((like) => {
+            if (like.item_id === elem.imdbid) {
+              btn.querySelector('span').innerHTML = like.likes;
+            }
+          });
+        });
+      }
+    });
+  });
```

Comment:
[Optional Changes]
You can consider moving this block of code to a new module to follow ES6 best practices.